### PR TITLE
[ID] Add "Beat" & "Music_theory/Beat"

### PR DESCRIPTION
**Pull request by mousesalt**
Add Indonesian translation of [Beat](https://osu.ppy.sh/wiki/en/Beat) & [Music_theory/Beat](https://osu.ppy.sh/wiki/en/Music_theory/Beat)

### CODE REVIEW

File: wiki/Beat/id.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+stub: true
+tags:
+  - ketukan
+  - garis putih
+  - garis-garis putih
+  - ketukan putih
+  - ketukan-ketukan putih
+  - keseluruhan ketukan
```
Comment (Nivalyx) [inline wiki/Beat/id.md:9]: `whole beat` -- given its musical term -- doesn't literally translate into `keseluruhan ketukan` tho...

it's more of a `satu ketukan` or `satu ketukan penuh` instead

File: wiki/Beat/id.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  - tick putih
+
+---
+
+# Ketukan (Beat)
+
+Sebuah **ketukan (beat)** adalah periode waktu di mana lagu-lagu dapat dibagi berdasarkan masing-masing [BPM](/wiki/Beatmapping/Beats_per_minute). Sebuah [beatmap](/wiki/Beatmap) dengan timing yang tepat akan memiliki setiap ketukan yang sesuai dengan ritme lagu, dan seluruh ketukan harus berada di antara dua garis putih di [timeline](/wiki/Beatmap_Editor/Timelines).
```
Comment (Nivalyx) [inline wiki/Beat/id.md:16]: `**Ketukan (beat)** adalah sebuah rentang waktu yang membagi lagu-lagu berdasarkan [BPM](/wiki/Beatmapping/Beats_per_minute)-nya masing-masing...`

File: wiki/Music_theory/Beat/id.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Ketukan (Beat)
+
+Sebuah **ketukan (beat)** adalah satuan waktu yang berulang dalam musik, khususnya denyut nadi musik yang diketuk oleh pendengar. Di osu!, kamu dapat membandingkannya dengan tanda tick putih di [editor timeline](/wiki/Beatmap_Editor/Timelines) yang menjadi kekuatan irama lagu.
+
+![Sebuah screenshot dalam permainan pada editor timeline](img/timeline.png "Semua tanda tick putih mewakili irama dalam musik")
```
Comment (Nivalyx) [inline wiki/Music_theory/Beat/id.md:5]: "Screenshot" should be translated as well for consistency i suppose

`Sebuah screenshot dalam permainan pada editor timeline` --> i.e. `Cuplikan kolom timeline pada editor`

File: wiki/Music_theory/Beat/id.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Ketukan (Beat)
+
+Sebuah **ketukan (beat)** adalah satuan waktu yang berulang dalam musik, khususnya denyut nadi musik yang diketuk oleh pendengar. Di osu!, kamu dapat membandingkannya dengan tanda tick putih di [editor timeline](/wiki/Beatmap_Editor/Timelines) yang menjadi kekuatan irama lagu.
```
Comment (Nivalyx) [inline wiki/Music_theory/Beat/id.md:3]: this translation feels very different w/ the other one you have tho, might as well making everything more consistent between the two

File: wiki/Beat/id.md (hunk 1 above)
Comment (mousesalt) [reply wiki/Beat/id.md:9]: is any better or common usage while we can adds more

Comment (Nivalyx) [reply wiki/Beat/id.md]: i don't think there's any other alternative (that i'm aware of, at least) tho...

however i'm *very* sure that the term `whole beat` doesn't translate into `keseluruhan ketukan`, yeah

Comment (mousesalt) [reply wiki/Beat/id.md]: i'd fixed this

File: wiki/Music_theory/Beat/id.md (hunk 4 above)
Comment (mousesalt) [reply wiki/Music_theory/Beat/id.md]: yeah it should.

**Review by Nivalyx — APPROVED** [review]
kk all should be good as far as i'm aware of 👍

**Review by ZeroPyrozen — APPROVED** [review]
All good 👍

**Comment by TicClick** [comment]
this doesn't look right, why are there two articles that cover essentially the same subject? we probably need to merge them (and I mean the originals first, the translations later).

**Comment by mousesalt** [comment]
we have discussion regarding similar subjects. the pronunciation is same mean each others.

**Comment by Nivalyx** [comment]
o @mousesalt that wasn't what TicClick meant by that comment

@mousesalt itu maksudnya TicClick "lho, ini aneh, kok di halaman wiki aslinya (yang bahasa inggris) bisa-bisanya sampai ada 2 artikel yang ngebahas hal yang sama begini. ini kayanya harus dibenerin dulu dari sananya deh"

**Comment by mousesalt** [comment]
@Nivalyx seep 
emang bingung dari pertama, kenapa ada dua artikel.

**Comment by mousesalt** [comment]
right, we will wait further information

File: wiki/Beat/id.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+tags:
+  - ketukan
+  - garis putih
+  - garis-garis putih
+  - ketukan putih
+  - ketukan-ketukan putih
+  - satu ketukan
+  - satu ketukan penuh
+  - tick putih
+
```
Comment (TicClick) [inline wiki/Beat/id.md:12]:
```suggestion
```

**Review by TicClick — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]
not going to hold you back any longer, this needs to be a separate decision which I can't really make right now

**Pull request merged**